src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -130,7 +130,7 @@ public function expiresAt($expiration): CacheItemInterface
     /**
      * {@inheritdoc}
      */
-    public function expiresAfter($time): CacheItemInterface
+    public function expiresAfter($time): static
```

Comment:
Can we also add argument types so we are aligned with the interface?

Reply:
added (needed to drop "psr/cache": "^1.0  for it to work)